### PR TITLE
Update changelog.md

### PR DESCRIPTION
**Pull request by nunomaduro**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: source/docs/changelog.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Changelog
```
Comment (nunomaduro) [inline source/docs/changelog.md:1]: @octoper this is not the good format tho. Can you double check?

Comment (owenvoke) [reply source/docs/changelog.md:1]: Just wondering, how do you want it displayed? 🙂 Guess it needs to be added to [the nav](https://github.com/pestphp/website/blob/master/navigation.php#L38) when it's ready.

Comment (octoper) [reply source/docs/changelog.md:1]: @nunomaduro Yeah you are right! I'm dump, I will fix it

Comment (alexmartinfr) [reply source/docs/changelog.md:1]: We could go with Tailwind's docs idea of having more separate sections:

```
Getting Started:

- Installation
- Release Notes


Using Pest:

- Writing Tests
- Underlying Test Case
- …
```

Capture d’écran 2020-06-05 à 10 56 59

**Pull request merged**